packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -391,11 +405,20 @@ class _CupertinoSheetTransitionState extends State<CupertinoSheetTransition> {
       reverseCurve: Curves.easeInToLinear,
       parent: widget.secondaryRouteAnimation,
     );
+    _upDragController = AnimationController(
```

Comment:
Nit: I find the `upDragController`, `downDragController` not immediately obvious. Would something like `stretchDragController` and `popDragController` be more clear?

Reply:
I see, it is definitely not linear.
As you pointed out, I have modified it to specify an easeOut for animateBack.

https://github.com/flutter/flutter/pull/168547/commits/7131376356c82684f6fa79ed2f5f6c8dcdd24425